Documentation/ceph-csi-drivers.md

```diff
@@ -83,30 +88,37 @@ statefulset.apps/csi-rbdplugin-attacher         1         1         1d
 statefulset.apps/csi-rbdplugin-provisioner      1         1         1d
 ```
 
-## Test the CSI driver
-
 Once the plugin is successfully deployed, test it by running the following example.
 
-### Create the StorageClass
+## Test RBD CSI Driver
+
+### Create RBD StorageClass
 
-This [storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml) expect a pool named `rbd` in your Ceph cluster. You can create this pool using [rook pool CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
+This
+[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml)
+expect a pool named `rbd` in your Ceph cluster. You can create this pool using
+[rook pool
+CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
 
 Please update `monitors` to reflect the Ceph monitors.
 
-### Create the Secret
+### Create RBD Secret
 
-Create a Secret that matches that specified in the [storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml).
+Create a Secret that matches that specified in the
```

Comment:
This is a bit difficult to read. I can't figure out what the secret should contain from this sentence alone.
Do you mean "Create a Secret that matches the namespace and name specified in the storageclass"?

Reply:
create a secret that matches the `adminid` and `userid` specified in storage class.